build.sbt

```diff
@@ -2579,12 +2579,19 @@ lazy val `engine-runner` = project
           )
       }
 
-      val exec =
-        s"$jlink --module-path ${modules.mkString(":")} --output $smallJdkDirectory --add-modules $NI_MODULES,$JDK_MODULES,$DEBUG_MODULES,$PYTHON_MODULES"
-      val exitCode = scala.sys.process.Process(exec).!
-
+      var jlinkArgs = Seq(
+        "--module-path",
+        modules.mkString(File.pathSeparator),
+        "--output",
+        smallJdkDirectory.toString(),
+        "--add-modules",
+        s"$NI_MODULES,$JDK_MODULES,$DEBUG_MODULES,$PYTHON_MODULES"
+      )
+      val exitCode = scala.sys.process.Process(jlink.toString(), jlinkArgs).!
```

Comment:
@jdunkerley reports https://github.com/enso-org/enso/actions/runs/9184711886/job/26530345747 failure on Mac Arm. It looks like a **space in path** issue and it should be fixed by this change.